analysis_options.yaml

```diff
@@ -21,7 +21,7 @@ linter:
   # or a specific dart file by using the `// ignore: name_of_lint` and
   # `// ignore_for_file: name_of_lint` syntax on the line or in the file
   # producing the lint.
-  rules:
+  rules:   
```

Comment:
what change?

Reply:
It's nothing. I enter something  and forgot to erase space.